src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimantResponseConfirmsToProceedLiPRespondentNotificationHandler.java

```diff
@@ -72,7 +73,7 @@ private String getEmailTemplate(CaseData caseData, boolean shouldSendEmailToDefe
                 ? notificationsProperties.getRespondentSolicitorNotifyToProceedInMediation()
                 : notificationsProperties.getRespondentSolicitorNotifyToProceedSpecWithAction();
         }
-        if (isClaimantNotProceedLipVsLRWithNoc(caseData)) {
+        if (isClaimantNotProcessLipVsLRWithNoc(caseData)) {
```

Comment:
Method name should be `isClaimantNotProceedLipVsLRWithNoc`.  I think "Process" was a typo.

Reply:
fixed.